.github/ISSUE_TEMPLATE/tracking.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "📊 Tracking Issue"
```

Comment:
How do we prevent people from outside the team from creating these, maybe because they think this is the type of issue they're creating for whatever reason. This is a possibility because the word 'tracking issue' can mean different things to different people.

Maybe we rename this file to tracking (internal team use only).md or something like that so that it shows up in the template chooser (is this how issue templates work?) or is there something in the config.yml file that can help us here.

Reply:
In the "about" header, which people see as a subtitle when picking an issue template, we say "for project management" but I can change it to "for internal use only"